irods/test/data_obj_test.py

```diff
@@ -2174,35 +2176,52 @@ def test_touch_operation_does_not_work_when_given_a_collection__525(self):
         with self.assertRaises(ex.InvalidInputArgument):
             user_session.data_objects.touch(home_collection_path)
 
+    def assert_redirect_happens(self, resource, open_opts):
```

Comment:
`resource` parameter appears to be unused here.

Consider adding a couple more words to indicate what this does. e.g. `assert_redirect_happens_on_open` or something like that?

Reply:
done